measures/TableMeasures/ScalarQuantColumn.h

```diff
@@ -194,6 +194,15 @@ template<class T> class ScalarQuantColumn
   // Throw an exception if the object is null.
   void throwIfNull() const;
 
+  // get the column as a Quantum<Vector<T> >. If <src>unit</src> is
+  // not null, the returned Quantum will have that unit. Else if
+  // itsConv is True, the returned Quantum will have itsUnitOut as
+  // the unit. Else if the units are variable, the
+  // values in the returned Vector have been converted to the
+  // unit of the 0th row entry. Otherwise, the units of the returned
+  // Quantum are itsUnit.
+  SHARED_PTR<Quantum<Vector<T> > > getColumn(const Unit* unit=NULL) const;
```

Comment:
using a empty unit as the default is nicer than a pointer to a unit

the documentation should not talk in terms of private variables, better would be for example "if unit is empty it will be converted to the unit associated to the column or the unit of the 0th row if `isUnitVariable()` is true"